### PR TITLE
OPT-479 add syslog-ng

### PR DESCRIPTION
**Pull request by Itsariandl**
# Notion Ticket
https://notion.so/OPT-479

# Pull request checklist

I have ...
- [x] assigned the Notion ticket to myself and update it's status
- [x] added the link to Notion ticket to the PR description and vice versa
- [x] tested the changes locally and described the testing results in the Notion ticket.
- [x] added `review` label to pull request, If the pull request is ready for review

### CODE REVIEW

**Comment by Itsariandl** [comment]
on this step we have syslog-ng , promtail and loki 

I deleted the mailhog and stunnel 
Screenshot 2024-10-25 at 12 34 36 PM

**Comment by mehr74** [comment]
The Notion ticket related to pull request is about adding syslogs. Removing mailhog and tunnel should be done over another pull request with appropriate notion ticket related to that.

File: linux-monitoring/docker-compose.yaml [hunk 1]
```diff
@@ -51,17 +54,37 @@ services:
...
+    image: grafana/promtail:latest
     ports:
-      - 465:465
+      - 1514:1514
+      - 9080:9080
+    command: -config.file=/etc/promtail/promtail-config.yaml
+    restart: always
+    volumes:
+    - ./promtail/promtail-config.yaml:/etc/promtail/promtail-config.yaml
+    networks:
```
Comment (mehr74) [inline linux-monitoring/docker-compose.yaml:77]: Do we need network loki?

**Review by mehr74 — CHANGES_REQUESTED** [review]
Do we need a network in docker-compose?

File: linux-monitoring/loki/loki-config.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        enabled: true
+        max_size_mb: 100
+
+query_scheduler:
+  max_outstanding_requests_per_tenant: 10000
+
+schema_config:
+  configs:
+  - from: 2020-10-24
+    store: boltdb-shipper
```
Comment (mehr74) [inline linux-monitoring/loki/loki-config.yaml:32]: The configuration you are using here is already depricated? Have you check the documentation?

File: linux-monitoring/loki/loki-config.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    schema: v11
+    index:
+      prefix: index_
+      period: 24h
+
+limits_config:
+  max_query_series: 5000
+
+ruler:
+  alertmanager_url: http://localhost:9093
```
Comment (mehr74) [inline linux-monitoring/loki/loki-config.yaml:43]: Is this url fine? I thought it should be alertmanager:9093

**Review by mehr74 — CHANGES_REQUESTED** [review]

File: linux-monitoring/loki/loki-config.yaml (hunk 3 above)
Comment (mehr74) [reply linux-monitoring/loki/loki-config.yaml]: fixed

**Review by mehr74 — APPROVED** [review]

**Pull request merged**